services/navigation/builtin/builtin.go

```diff
@@ -412,17 +443,23 @@ func (svc *builtIn) startWaypoint(ctx context.Context, extra map[string]interfac
 					svc.logger.Infof("skipping waypoint %+v since it was deleted", wp)
 					continue
 				}
+				svc.logger.Warnf("failed to navigate to waypoint: %+v due to error %s", wp, err.Error())
```

Comment:
super nit but looks like the colon is in the wrong place. In general a lot of this logging would be cleaned up with calls to `errors.Wrap` so that you don't have to do this manually

Reply:
Could also move this into an `else` since we're continuing on both paths now

Reply:
done